tubular/scripts/retire_one_learner.py

```diff
@@ -23,28 +24,25 @@
 """
 
 
+import logging
+import sys
 from functools import partial
 from os import path
 from time import time
-import logging
-import sys
 
 import click
-from six import text_type
 from requests.exceptions import HTTPError
+from six import text_type
```

Comment:
Do you think we still need the `six` library in our codebase?

Reply:
removed

Reply:
And why it is still imported and required?